src/core/opamStd.mli

```diff
@@ -510,11 +510,12 @@ module Sys : sig
   (** Queried lazily *)
   val os: unit -> os
 
-  (** The output of the command "uname", with the given argument. Memoised. *)
-  val uname_cmd: string -> string option
+  (** The output of the command "uname -U". FreeBSD only. *)
+  val uname_freebsd_version: unit -> string option
```

Comment:
It's not just uname, but uname -U to retrieve freebsd version. Maybe something like "freebsd_version"

Reply:
`freebsd_version` doesn't give the impression that a command is going to be called so i'd rather keep the current name

Reply:
`uname` gives the impression that a command is called but it is not the case. This kind of information should be in the documentation.

Reply:
```suggestion
  val get_freebsd_version: unit -> string option
```
As discussed on dev meeting